packages/openneuro-server/src/libs/authentication/jwt.js

```diff
@@ -153,10 +154,13 @@ export const authenticate = (req, res, next) => {
 }
 
 export const authSuccessHandler = (req, res, next) => {
+  const redirectPath = req.query.state 
+    ? atob(req.query.state)
```

Comment:
You can avoid adding atob (unmaintained package and obsolete in Node 16) like this:
```suggestion
    ? Buffer.from(req.query.state, 'base64').toString()
```